app/models/podcast_episode.rb

```diff
@@ -86,6 +86,14 @@ def published
     true
   end
 
+  def readable_publish_date
+    published_at.strftime("%b %e")
+  end
+
+  def published_timestamp
+    published_at.utc.iso8601
+  end
+
```

Comment:
These should go into the `podcast_episode_decorator.rb` file

Reply:
thanks @rhymes for helps, i'm working on the issue.

Reply:
Done! thanks @rhymes